sysutils/libvirt/Portfile

```diff
@@ -29,14 +30,11 @@ homepage            https://libvirt.org
 master_sites        ${homepage}/sources/
 use_xz              yes
 
-depends_build       port:autoconf \
-                    port:automake \
-                    port:bash-completion \
-                    port:libtool \
-                    port:pkgconfig \
-                    port:python38 \
-                    port:py38-docutils \
-                    port:perl5
+depends_build-append    port:bash-completion \
+                        port:pkgconfig \
+                        port:python38 \
```

Comment:
python39 is now the default version in MacPorts. If this port will work with python39 then it should be switched to do so. If you don't want to do it as part of this commit I can do it later.

I would like to keep the alignment of this column as it was before. I suggest:

```
depends_build-append \
                    port:bash-completion \
                    port:pkgconfig \
                    ...
```